tests/events/18-suite-event-mail.t

```diff
@@ -16,6 +16,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
 # Test event mail.
+CYLC_TEST_IS_GENERIC=false
```

Comment:
I think the `mail` command is not sending to the port that smtpd.DebuggingServer is on - any idea how that is configured?